x-pack/test/functional/apps/ml/anomaly_detection/anomaly_explorer.ts

```diff
@@ -126,6 +137,154 @@ export default function ({ getService }: FtrProviderContext) {
+          );
+
+          await ml.testExecution.logTestStep('clears the selection');
+          await ml.anomalyExplorer.clearSwimLaneSelection();
+          await ml.navigation.assertCurrentURL(
+            "/app/ml/explorer?_g=(ml%3A(jobIds%3A!(fq_multi_1_ae))%2CrefreshInterval%3A(display%3AOff%2Cpause%3A!t%2Cvalue%3A0)%2Ctime%3A(from%3A'2016-02-07T00%3A00%3A00.000Z'%2Cto%3A'2016-02-11T23%3A59%3A54.000Z'))&_a=(explorer%3A(mlExplorerFilter%3A()%2CmlExplorerSwimlane%3A(viewByFieldName%3Aairline%2CviewByFromPage%3A1%2CviewByPerPage%3A10))%2Cquery%3A(query_string%3A(analyze_wildcard%3A!t%2Cquery%3A'*')))"
+          );
+          await ml.anomalyExplorer.assertClearSelectionButtonVisible(false);
+          await ml.anomaliesTable.assertTableRowsCount(25);
+        });
+
+        it('allows to change the swim lane pagination', async () => {
+          await ml.testExecution.logTestStep('checks default pagination');
+          await ml.swimLane.assertPageSize(viewBySwimLaneTestSubj, 10);
+          await ml.swimLane.assertActivePage(viewBySwimLaneTestSubj, 1);
+
+          await ml.testExecution.logTestStep('updates pagination');
+          await ml.swimLane.setPageSize(viewBySwimLaneTestSubj, 5);
+
+          const axisLabels = await ml.swimLane.getAxisLabels(viewBySwimLaneTestSubj, 'y');
+          expect(axisLabels.length).to.eql(5);
+
+          await ml.swimLane.selectPage(viewBySwimLaneTestSubj, 3);
+
+          await ml.testExecution.logTestStep('resets pagination');
+          await ml.swimLane.setPageSize(viewBySwimLaneTestSubj, 10);
+          await ml.swimLane.assertActivePage(viewBySwimLaneTestSubj, 1);
+        });
+
+        it('support cell selection by click on View By swim lane', async () => {
+          await ml.testExecution.logTestStep('clicks on the View By swim lane cell');
+          await ml.anomalyExplorer.assertSwimlaneViewByExists();
+          const sampleCell = (await ml.swimLane.getCells(viewBySwimLaneTestSubj))[0];
+
+          await ml.swimLane.selectSingleCell(viewBySwimLaneTestSubj, {
+            x: sampleCell.x + cellSize,
+            y: sampleCell.y + cellSize,
+          });
+
+          await ml.swimLane.assertSelection(viewBySwimLaneTestSubj, {
+            x: [1454817600000, 1454832000000],
+            y: ['AAL'],
+          });
+
+          await ml.testExecution.logTestStep('highlights the Overall swim lane');
+          await ml.swimLane.assertSelection(overallSwimLaneTestSubj, {
+            x: [1454817600000, 1454832000000],
+            y: ['Overall'],
+          });
+
+          await ml.testExecution.logTestStep('clears the selection');
+          await ml.anomalyExplorer.clearSwimLaneSelection();
```

Comment:
Would be good to do some additional checks similar to what's done above for the overall swim lane: check that charts are displayed, that influencers list and anomalies list changed and the the URL state changed. And the same after `clearSwimLaneSelection`.